activerecord/lib/active_record/autosave_association.rb

```diff
@@ -91,8 +91,10 @@ module ActiveRecord
   #   post.save # => saves both post and comment
   #
   #   post = Post.create(title: 'ruby rocks')
-  #   post.comments.create(body: 'hello world')
-  #   post.save # => saves both post and comment
+  #   comment = post.comments.create(body: 'hello world')
+  #   post.title += "!"
+  #   comment.body += "!"
```

Comment:
I think assigning a new value here, to match the other example below, would be clearer:

```suggestion
  #   comment.body = 'hi everyone'
```